www/content/tutorial.md

```diff
@@ -48,7 +48,7 @@ So far, so good!
 
 Try typing this in the REPL and pressing Enter:
 
-<samp class="repl-prompt">"Hello, World!"</samp>
+<pre><samp class="repl-prompt">"Hello, World!"</samp></pre>
```

Comment:
Without pre this was a different color (in light mode) compared to other code snippets.